packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -849,6 +854,14 @@ class _PopupMenuRoute<T> extends PopupRoute<T> {
     return super.createAnimation();
   }
 
+  void scrollTo(int selectedItemIndex) {
+    WidgetsBinding.instance.addPostFrameCallback((_) {
```

Comment:
I'm not sure what the distinction between this and SchedulerBinding.instance.addPostFrameCallback is. Most of our code appears to use the SchedulerBinding version.